src/Common/Commands.Common.Graph.RBAC/ActiveDirectory/ActiveDirectoryClient.cs

```diff
@@ -410,9 +410,9 @@ public Guid GetObjectId(ADObjectFilterOptions options)
             return principalId;
         }
 
-        public void UpdateApplication(string appObjectId, ApplicationUpdateParameters parameters)
+        public void UpdateApplication(Guid appObjectId, ApplicationUpdateParameters parameters)
```

Comment:
So, do we need these changes in this branch, or can we just add a dependency to the latest Profile?

If we do need them, when testing, can we simply use the latest version of profile from the gallery, rather than the version that is built here?

Reply:
The dependency on the latest version of Profile is sufficient and these changes shouldn't need to be in any other branch (unless we want to go back and patch the last version of 5.x.x). I used the latest version of Profile from the gallery when testing with these bits and confirmed the behavior is consistent with version 5.0.0 of Resources.